.drone.yml

```diff
@@ -95,7 +95,7 @@ steps:
       api_key:
         from_secret: github_public_repo
       files:
-        - /
+        - *
```

Comment:
What did this fix?

Reply:
I posted about it in Slack. See https://drone.spaceship.0x.org/0xProject/0x-mesh/748/3/1.

Reply:
Also https://github.com/0xProject/0x-mesh/pull/715